dev-client/src/translations/index.ts

```diff
@@ -17,22 +17,42 @@
 
 import {initReactI18next} from 'react-i18next';
 
+import {getLocales} from 'expo-localization';
+
 import i18n from 'i18next';
 
 import en from 'terraso-mobile-client/translations/en.json';
+import es from 'terraso-mobile-client/translations/es.json';
+
+export const fallbackLanguage = 'en';
+export const fallbackLocale = 'en-US';
+
+let deviceLanguage = getLocales()[0]?.languageCode;
```

Comment:
Question: I forget if/where we landed on MVP behavior as a team -- is this the final behavior that Derek and Courtney agreed to, or were we considering adding additional logic for how a user would specify their language of choice (for MVP or later)?

Reply:
We're starting without in-app UI for switching.